### DISCUSSION TITLE
42seoul.gitbook.io 어디까지 봤는지 알려주기

### DISCUSSION THREAD

**Discussion by get6** [42webserv/42webserv#9] [Study]
각자의 이해도가 얼마나 되는지 확인하기 위해서 기간을 정해두고 각자 읽어서 학습하는 시간이 필요할 것 같습니다.

다들 기간 내에 어디까지 읽었는지 댓글로 알려주세요. 제 기억으론 이번주까지 학습 부분 다 읽기 같았는데 맞나요?

**Comment by get6** [comment]
04.14 (금) 학습까지 완료

**Comment by gtd9511** [comment]
4.14 (금) 학습까지 빠르게 한번 훑은 뒤에 학습(웹) 부분 정리하면서 읽는중입니다.
